sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/MetricExporterConfiguration.java

```diff
@@ -172,7 +172,7 @@ private static void configurePrometheusMetrics(
     } catch (IOException e) {
       throw new IllegalStateException("Failed to create Prometheus server", e);
     }
-    Runtime.getRuntime().addShutdownHook(new Thread(server::stop));
+    Runtime.getRuntime().addShutdownHook(new Thread(server::close));
```

Comment:
Is this the same thing, just renamed, or a different function?

Reply:
Yeah - previous one is deprecated and our build prevents using deprecated stuff.

Reply:
It does make me wonder how to release autoconfigure as stable with the dependency on this pre-1.0 Prometheus artifact which is known to break some times. We may need to go ahead and implement our own endpoint...